js/blocks.js

```diff
@@ -3616,8 +3616,8 @@ function Blocks(canvas, stage, refreshCanvas, trashcan, updateStage, getStageSca
         if (myBlock.name === 'start' || myBlock.name === 'drum') {
             turtle = myBlock.value;
             var turtleNotInTrash = 0;
-            for(var i = 0; i < this.turtles.turtleList.length; i++) {
-                if(!blocks.turtles.turtleList[i].trash) {
```

Comment:
I agree that blocks should be this here for clarity, but what I don't understand is why it doesn't work with blocks since blocks.turtles should be the same as this.turtles